admin/server/alerts.go

```diff
@@ -577,6 +599,13 @@ type alertYAML struct {
 	Intervals struct {
 		Duration string `yaml:"duration"`
 	} `yaml:"intervals"`
+	Data struct {
+		Connector  string         `yaml:"connector"`
+		SQL        string         `yaml:"sql"`
+		MetricsSQL string         `yaml:"metrics_sql"`
+		API        string         `yaml:"api"`
+		Args       map[string]any `yaml:"args"`
+	} `yaml:"data"`
```

Comment:
You could also consider just having `Data map[string]any` and relying on the frontend to provide a valid resolver? It's a bit less safe, but reduces duplication of the resolver definition.